airbyte-integrations/connectors/source-iterable/source_iterable/source.py

```diff
@@ -53,101 +42,68 @@
-        if all_streams_accessible():
-            streams.extend(
-                [
-                    Users(authenticator=authenticator, **date_range),
-                    ListUsers(authenticator=authenticator),
-                    EmailBounce(authenticator=authenticator, **date_range),
-                    EmailClick(authenticator=authenticator, **date_range),
-                    EmailComplaint(authenticator=authenticator, **date_range),
-                    EmailOpen(authenticator=authenticator, **date_range),
-                    EmailSend(authenticator=authenticator, **date_range),
-                    EmailSendSkip(authenticator=authenticator, **date_range),
-                    EmailSubscribe(authenticator=authenticator, **date_range),
-                    EmailUnsubscribe(authenticator=authenticator, **date_range),
-                    PushSend(authenticator=authenticator, **date_range),
-                    PushSendSkip(authenticator=authenticator, **date_range),
-                    PushOpen(authenticator=authenticator, **date_range),
-                    PushUninstall(authenticator=authenticator, **date_range),
-                    PushBounce(authenticator=authenticator, **date_range),
-                    WebPushSend(authenticator=authenticator, **date_range),
-                    WebPushClick(authenticator=authenticator, **date_range),
-                    WebPushSendSkip(authenticator=authenticator, **date_range),
-                    InAppSend(authenticator=authenticator, **date_range),
-                    InAppOpen(authenticator=authenticator, **date_range),
-                    InAppClick(authenticator=authenticator, **date_range),
-                    InAppClose(authenticator=authenticator, **date_range),
-                    InAppDelete(authenticator=authenticator, **date_range),
-                    InAppDelivery(authenticator=authenticator, **date_range),
-                    InAppSendSkip(authenticator=authenticator, **date_range),
-                    InboxSession(authenticator=authenticator, **date_range),
-                    InboxMessageImpression(authenticator=authenticator, **date_range),
-                    SmsSend(authenticator=authenticator, **date_range),
-                    SmsBounce(authenticator=authenticator, **date_range),
-                    SmsClick(authenticator=authenticator, **date_range),
-                    SmsReceived(authenticator=authenticator, **date_range),
-                    SmsSendSkip(authenticator=authenticator, **date_range),
-                    SmsUsageInfo(authenticator=authenticator, **date_range),
-                    Purchase(authenticator=authenticator, **date_range),
-                    CustomEvent(authenticator=authenticator, **date_range),
-                    HostedUnsubscribeClick(authenticator=authenticator, **date_range),
-                    Events(authenticator=authenticator),
-                ]
-            )
+        streams.extend(
```

Comment:
Is there a plan to move these to low code eventually? If not, why?

Reply:
Makes sense. Can you add a comment here about that?

Reply:
added